README.md

```diff
@@ -30,6 +30,7 @@ selector = Selective(SelectionMethod.Correlation(threshold=0.5, method="pearson"
 selector = Selective(SelectionMethod.Statistical(num_features=3, method="anova"))
 selector = Selective(SelectionMethod.Linear(num_features=3, regularization="none"))
 selector = Selective(SelectionMethod.TreeBased(num_features=3))
+selector = Selective(SelectionMethod.TextBased(num_features=3))
 
 # Feature reduction
 subset = selector.fit_transform(data, label)
```

Comment:
now.. this example with this data/labal is not gonna work with the TextBased selector, no?

Reply:
I think we can create a separate section after Benchmarking to show an example of TextBased selection and make it explicit what data and labels look like in the input with a small toy problem

Reply:
Yes, it does not work. Great Idea! will add a section after Benchmarking and will let you to review.

Reply:
The section added. Please let me know with your feedback.

Reply:
then you want to delete the above line, no?

Reply:
I added a reference for the sub-section text-based selective in Selective: Feature Selection Library. Since people first take a look at Quick Start and may lose where the one for text-based is added.